src/app/features/activity-list/components/transaction-list/bitcoin-transaction/bitcoin-transaction-icon.tsx

```diff
@@ -0,0 +1,56 @@
+import { FiArrowDown as IconArrowDown, FiArrowUp as IconArrowUp } from 'react-icons/fi';
+
+import { Box, BoxProps, Circle, ColorsStringLiteral, Flex, color } from '@stacks/ui';
+
+import { BitcoinTransaction } from '@shared/models/transactions/bitcoin-transaction.model';
+
+import { BtcIcon } from '@app/components/icons/btc-icon';
+import { useCurrentBtcNativeSegwitAccountAddressIndexZero } from '@app/store/accounts/blockchain/bitcoin/native-segwit-account.hooks';
+
+import { isBitcoinTxInbound } from './bitcoin-transaction.utils';
+
+interface TransactionIconProps extends BoxProps {
+  transaction: BitcoinTransaction;
+}
+type BtcStatusColorMap = Record<string, ColorsStringLiteral>;
```

Comment:
Do we have types anywhere else we could use for this? Otherwise we could type as `'pending' | 'success'`